proto/tendermint/services/version/v1/version.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package tendermint.services.version.v1;
```

Comment:
I rather meant reusing the message types already defined in the `tendermint.version` package, to be renamed into `cometbft.version.v1`.

Reply:
The message types already defined in `tendermint.version` aren't sufficient, from what I can see, to provide this information. So we'd have to add a structure there.

Reply:
> for someone trying to make sense of the entire assembly of protos it would be difficult to understand the purposes of `cometbft.version` vs `cometbft.services.version`

I'm not sure why the types in `tendermint.version` were separate out from `tendermint.types` in the first place. Anything under `{tendermint|cometbft}.services` should be considered to be related to the gRPC services we expose.

The `tendermint.state.Version` type is actually almost the same as what we'd potentially need from this version service: https://github.com/cometbft/cometbft/blob/f667d3fcab8c961ae3921a8d53254c6cfc0a824b/proto/tendermint/state/types.proto#L56-L59

It's just missing the P2P version. And I could add the application version to this version service response.

It still makes sense to me to "denormalize" the data returned by this version service for ergonomic and sustainability reasons. Decoupling the response field as much as possible from other types will allow for better decoupling of the gRPC interface's evolution from the core types used within data structures like the block header.